File: console/src/features/projects/dto/project-response.ts
export interface ProjectResponse {
  id: string;
  account_id: string;
  name: string;
  description?: string;
  monthly_points_supply?: number;
  monthly_token_supply?: number;
  exchange_ratio?: number;
  token_value?: number;
  status: string;
  created_at: number;
  updated_at: number;
}

File: app/src/features/projects/dto/project_response.rs
use crate::common::types::Partition;
use crate::features::projects::ProjectStatus;
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq)]
pub struct ProjectResponse {
    pub id: String,
    pub account_id: Partition,
    pub name: String,
    pub description: Option<String>,
    pub monthly_token_supply: i64,
    pub status: ProjectStatus,
    pub created_at: i64,
    pub updated_at: i64,
}
